packages/react-charts/src/components/ChartUtils/chart-padding.ts

```diff
@@ -1,17 +1,14 @@
-import { get, isEmpty, isFinite } from 'lodash';
 import { PaddingProps } from 'victory-core';
 
 export const getPaddingForSide = (
   side: 'bottom' | 'left' | 'right' | 'top',
   padding: PaddingProps,
   fallback: PaddingProps
 ): number => {
-  if (!isEmpty(padding)) {
-    return get(padding, side, 0);
-  }
-
-  if (isFinite(padding)) {
+  if (typeof padding == 'number') {
     return padding as number;
+  } else if (typeof padding == 'object') {
+    return padding[side] || 0;
   }
```

Comment:
I'm not sure this change is really the equivalent of what we had in place? We want to ensure padding is truly empty, not just an object, and there may be one or more properties.

Reply:
Fixed!